rest/model/perf.go

```diff
@@ -0,0 +1,200 @@
+package model
+
+import (
+	dbmodel "github.com/evergreen-ci/sink/model"
+	"github.com/pkg/errors"
+)
+
+type APIPerformanceResult struct {
+	Name         APIString              `json:"name"`
+	Info         APIPerformanceResultID `json:"info"`
+	CreatedAt    APITime                `json:"create_at"`
+	CompletedAt  APITime                `json:"completed_at"`
+	Version      int                    `json:"version"`
+	Source       []APIArtifactInfo      `json:"source_info"`
+	AuxilaryData []APIArtifactInfo      `json:aux_data"`
+	Total        *APIPerformancePoint   `json:"total"`
```

Comment:
Why are these pointers?

Reply:
Because they are pointers in `PerformanceResult` https://github.com/evergreen-ci/sink/blob/master/model/perf.go#L45-L47

Reply:
should probably make it omit empty then.